third-party/dav1d.cmd

```diff
@@ -21,3 +21,8 @@ meson build --default-library=static --buildtype release --prefix "$(pwd)/dist"
 ninja -C build
 ninja -C build install
 cd ..
+
+echo ""
+echo "----- NOTE ----"
+echo "Please add the path to the pkg-config file to your PKG_CONFIG_PATH, like this:"
+echo "export PKG_CONFIG_PATH=\$PKG_CONFIG_PATH:$(pwd)/dav1d/dist/lib/x86_64-linux-gnu/pkgconfig"
```

Comment:
Hi Brad,

I copied this path from the comment at line 4. This path will only work on Linux x86_64. I can clarify that in the message. How about this?

> Please add the path to the pkg-config file to your PKG_CONFIG_PATH. For example, on Linux x86_64, run:
> export PKG_CONFIG_PATH=\$PKG_CONFIG_PATH:$(pwd)/dav1d/dist/lib/x86_64-linux-gnu/pkgconfig

Reply:
That seems better.

Reply:
Done.